static/src/js/components/SearchPanels/SearchPanels.js

```diff
@@ -280,6 +280,8 @@ class SearchPanels extends PureComponent {
     ]
 
     const setCollectionSort = (value) => {
+      if (value === activeCollectionsSortKey) return
+
```

Comment:
Is there a test case in `searchPanels.test.js` for this? If not, it'd be good to add one.

Reply:
Hey so after looking over this extensively, I am unsure how to get the children components to render in such a way that I may `simulate('mouseOver')` and show the dropdown then `simulate('click')` and assert the functions do or do not call. 

The showing and clicking of the sort drop down seems to be taken care of here, 

https://github.com/nasa/earthdata-search/blob/584ba390b60b828c0895c331557ded14cc9ad899/static/src/js/components/Panels/__tests__/PanelGroupHeader.test.js#L158

What do you think I should/can do?

Reply:
If no one has mentioned it yet, this might be a good opportunity to rewrite the test for this file in testing library to move it away from enzyme

Reply:
I hadn't mentioned it just because the change was so small. That being said it sure would be easier to work with and we have to do it anyway